### ISSUE TITLE
import error when json file has clones

### ISSUE THREAD

**Issue by fullyint** [WuTheFWasThat/vimflowy#180]
Thank you for creating vimflowy. I love it! I'm running version 290c885 on local/dev.

Importing a json file with clones produces `An error was caught. Please refresh the page to avoid weird state.` I couldn't quite tell if this was an instance of the problem of #105 or whether that was resolved.

I regret that I'm a js newb and couldn't easily pull together a quality PR. However, the problem seems to be that when the import processes the clone, [`id_mapping`](https://github.com/WuTheFWasThat/vimflowy/blob/290c885/assets/js/document.coffee#L482) will be empty because no `id_mapping` is passed [when `loadTo` is called](https://github.com/WuTheFWasThat/vimflowy/blob/290c885/assets/js/mutations.coffee#L213). I began to wonder if `id_mapping` belongs in the DataStore. 

**Steps to reproduce**
Consider this simplified test outline:

- item
  - clone
- clone

Here is the json file exported:
```
{
  "text": "",
  "children": [
    {
      "text": "item",
      "children": [
        {
          "text": "clone",
          "id": 94
        }
      ]
    },
    {
      "clone": 94
    }
  ]
}
```
Here is an excerpt from the console log after attempting to import the json above:
```
setting local storage test6save:95:parent [94]
getting from local storage test6save:95:detached_parent null
got nothing, defaulting to null
getting from local storage test6save:lastSave 0
got  1464628495152
setting local storage test6save:95:line ["c", "l", "o", "n", "e"]
getting from local storage test6save:lastSave 0
got  1464628495176
setting local storage test6save:95:collapsed undefined 
Caught error: 'Uncaught Error: Assertion error: assert error 
Expected false to be true' from  http://localhost:8080/js/bundle.js:3744 
Error:  g {stack: "Error↵    at Error.g (http://localhost:8080/js/bun…load (http://localhost:8080/js/bundle.js:4114:16)", message: "Assertion error: assert error↵Expected false to be true"} Error 
   at Error.g (http://localhost:8080/js/bundle.js:3706:18) 
   at Object.exports.assert (http://localhost:8080/js/bundle.js:3744:11) 
   at Document.loadTo (http://localhost:8080/js/bundle.js:3628:14) 
   at AddBlocks.mutate (http://localhost:8080/js/bundle.js:6146:30) 
   at Session.do (http://localhost:8080/js/bundle.js:7117:14) 
   at Session.addBlocks (http://localhost:8080/js/bundle.js:7741:22) 
   at Session.importContent (http://localhost:8080/js/bundle.js:6977:12) 
   at http://localhost:8080/js/bundle.js:4189:23    
   at FileReader.reader.onload (http://localhost:8080/js/bundle.js:4114:16) 
Uncaught Error: Assertion error: assert error 
Expected false to be true 
```

**Comment by WuTheFWasThat** [comment]
thanks for reporting this.  fixed in 50a83633fcc1067a56354335dddb98f5144bafa4

that piece of code is not so great, `id_mapping` is a bit of a hack

**Comment by WuTheFWasThat** [comment]
@fullyint feel free to reopen if this didn't solve your problem, of course
